ecllibrary/std/Crypto.ecl

```diff
@@ -84,17 +84,43 @@ EXPORT SymmetricEncryption(VARSTRING algorithm, VARSTRING passphrase) := MODULE
     END;
 END; // SymmetricEncryption module
 
-
-
+/**
+ * Encryption module containing symmetric encryption/decryption functions
+ *
+ * @param   algorithm      Symmetric algorithm to use, as returned by SupportedSymmetricCipherAlgorithms()
+ * @param   passphrase     Passphrase buffer to use for encryption/encryption
```

Comment:
minor name doesn't match parameter name.  I think I prefer the name here - it isn't really a buffer (that's more an implementation detail).

Reply:
Removed "buffer"